packages/desktop-client/src/components/PrivacyFilter.tsx

```diff
@@ -0,0 +1,109 @@
+import React, {
+  useState,
+  useCallback,
+  Children,
+  type ComponentPropsWithRef,
+  type ReactNode,
+  type ReactElement,
+} from 'react';
+
+import { usePrivacyMode } from 'loot-core/src/client/privacy';
+
+import useFeatureFlag from '../hooks/useFeatureFlag';
+
+import { View } from './common';
+
+export type ConditionalPrivacyFilterProps = {
+  children: ReactNode;
+  privacyFilter?:
```

Comment:
Do you think there could be a way to pass in the prop override the component wants without the need to manually write this function in each component?

Reply:
Maybe we can use react context for this?

Reply:
I was thinking something along the lines of `privacyFilter` providing an object that is merged with the default options?

Reply:
Updated privacyFilter to either be a boolean flag or `PrivacyFilterProps` which will be used to override the default props.